Sources/ConfidenceProvider/ConfidenceFeatureProvider.swift

```diff
@@ -50,6 +78,8 @@ public class ConfidenceFeatureProvider: FeatureProvider {
         }
 
         if self.initializationStrategy == .activateAndFetchAsync {
+            // TODO: Set the entire context
+            confidence?.context = ["targeting_key": "CACHED"]
```

Comment:
Perhaps we can remove this from this PR, adding it back when the full context can be properly passed along.
Also note that in this particular example, we will need to fetch the context from storage (also something for future PRs)

Reply:
Not really sure what we're trying to achieve here or why we're doing this? Why can't we set the whole context right away?

Reply:
If we want to avoid having the OF's `Value` and `EvaluationContext` types in the Confidence library (i.e. keep the Confidence library OpenFeature-agnostic), we first need to define some other types and convert the OF Context. I am keeping this for a followup PR